### PR TITLE
Other providers login feature

### PR DESCRIPTION
**Pull request by kiprot**
- create component to enable users to change their language/username when they want
- make some small changes to the login UI

### CODE REVIEW

File: src/main/ngapp/src/app/change-user-info/change-user-info.component.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    };
+    console.log("this is the to-put user", this.user);
+    
+    // PUT request to database.
+    const promise = this.http.put<LocalUser>("/user", this.user).toPromise();
+    console.log("put successfully performed for", this.user);
+    promise.then(response => {
+      console.log("User post request: ", response);
+      this.router.navigate(['/chat']);
+    });
```
Comment (MohamedShatry) [inline src/main/ngapp/src/app/change-user-info/change-user-info.component.ts:43]: Would you mind adding this method to chat data service? That way all http calls are made from the same service?

Comment (kiprot) [reply src/main/ngapp/src/app/change-user-info/change-user-info.component.ts:43]: good idea.
will do it with the next pr?

Comment (kiprot) [reply src/main/ngapp/src/app/change-user-info/change-user-info.component.ts:43]: will also add the POST request in chat-service (the one i used in select-language-component)

Comment (MohamedShatry) [reply src/main/ngapp/src/app/change-user-info/change-user-info.component.ts:43]: cool

**Pull request merged**